TUM Campus App/PlansTableViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  PlansTableViewController.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 16.12.16.
+//  Copyright © 2016 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class PlansTableViewController: UITableViewController {
+    
+    // MARK: - Table view data source
+    lazy var plans = Plans()
+    let loadingView = UIView()
+    let spinner = UIActivityIndicatorView()
+    let loadingLabel = UILabel()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.downloadData()
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return plans.getPlans().count
+    }
+
+    // load data into the rows
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let plan = plans.getPlan(forIndex: indexPath.row)
+        
+        let cell = tableView.dequeueReusableCell(withIdentifier: "planCell", for: indexPath) as! PlansTableViewCell
+
+        // Configure the cell...
+        cell.planTitleLabel.text = plan.title
```

Comment:
Not really since this is still a really controversial topic today. But you can see it in the rest of the app: https://github.com/TCA-Team/iOS/blob/master/TUM%20Campus%20App/TuitionCardTableViewCell.swift

As you can see there's an extra Object for the cell that is optimized for displaying it.
BTW: I know there's a setter here. I haven't updated some of the old code to the new style guidelines ;)